metapool/metapool.py

```diff
@@ -198,6 +198,7 @@ def calculate_norm_vol(dna_concs, ng=5, min_vol=2.5, max_vol=3500,
     sample_vols : numpy array of float
         The volumes to pool (nL)
     """
+    # TODO: possible divide by zero warnings.
```

Comment:
Will do! I encountered a divide by zero error after running some stock data through it - I believe it was because the volume was set to zero for blanks. I'll add it as an issue and remove the TODO.